appveyor.yml

```diff
@@ -1,31 +0,0 @@
-environment:
```

Comment:
Did we give up on automated Windows testing (AppVeyor) altogether? Or is this temporary?

Reply:
This was a mistake, I have since reverted.